main.go

```diff
@@ -133,7 +134,10 @@ func runXcodeBuildCmd(useStdOut bool, args ...string) (string, int, error) {
 
 	log.Printf("$ %s", cmdArgsForPrint)
 
-	err := buildCmd.Run()
+	var err error
+	progress.SimpleProgress(".", 2*time.Minute, func() {
```

Comment:
only use progress when useStdOut==true

Reply:
I mean `false`

Reply:
also useStdOut a little misunderstandable as stdout will be used but not in the command but for the dots